src/link/SpirV.zig

```diff
@@ -163,7 +163,7 @@ pub fn updateExports(
             .Vertex => spec.ExecutionModel.Vertex,
             .Fragment => spec.ExecutionModel.Fragment,
             .Kernel => spec.ExecutionModel.Kernel,
-            else => unreachable,
+            else => return,
```

Comment:
When does this happen?

Reply:
there are a few functions with C callconv in `test/behavior/`